data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+	job
+	repeat
+	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
+	cargo "secure package" 1
+	to offer
+		random < 20
+		not "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		"secure deep package" ++
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+
+mission "Deep Security Package Reveal"
+	minor
+	invisible
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	to offer
+		"reputation: Deep Security" >= 100
+		"secure deep package" >= 30
+		"mystery cube" >= 5
+		not "deep security reveal"
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log "Minor People" "Agent Wayde" `A young agent in the logistics division of Deep Security, he vouched for your increased job opportunities after you helped him out.`
+		conversation
+			`As you walk away from a food stand in the spaceport a rather young man in active wear steps away from the wall and smoothly sidles up next to you between one step and the next.`
+			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`
+			`	His words come out very quickly as if they'd been heavily rehearsed and, looking around, no one appears to be paying any attention to either of you.`
+			choice
+				`	"Sure. Who are you?"`
+					goto who
+				`	"Sorry, but I'm busy right now."`
+			`	"Uh, I was coming to thank you if that helps?" he says.`
+			choice
+				`	"Oh. What is it?"`
+				`	"Sorry, maybe next time."`
+					decline
+			
+			label who
+			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
+			choice
+				`	"Work is work."`
+					goto blindacceptance
+				`	"Ah, yes."`
+					goto blindacceptance
+				`	"Do all the secure packages belong to Deep Security?"`
+			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
+			label blindacceptance
+			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`
```

Comment:
```suggestion
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion and it seems like you qualified. You passed the background check at any rate.`
```

Reply:
I feel like @W1zrad would come along and tell me to take that one back out again?

Reply:
That's a spot where it's needed - you got two independent clauses so you need a comma. You _could_ stretch to keep it in since it's dialogue though.

Reply:
The comma wizard has spoken!